resources/views/emails/new_registration.blade.php

```diff
@@ -0,0 +1,11 @@
+@component('mail::message')
+    <h1>@lang('mail.dear') {{ $recipient }}!</h1>
+    <p>
+        @if ($is_collegist)
+            @lang('mail.new_registration_collegist', ['registered_user' => $registered_user])
+        @else
+            @lang('mail.new_registration_tenant', ['registered_user' => $registered_user])
+        @endif
+    </p>
```

Comment:
Add some basic data (status, email, etc.) and a button with a link the corresponding page.